lndmanage/lib/liquidityhints.py

```diff
@@ -333,3 +432,29 @@ def __repr__(self):
             for k, v in self._liquidity_hints.items():
                 string += f"{k}: {v}\n"
         return string
+
+    def extend_with_mission_control(self, mc_pairs):
+        logger.info("> Syncing mission control data.")
+        for pair in mc_pairs:
+            node_from = pair.node_from.hex()
```

Comment:
`node_from = NodeID(pair.node_from.hex())`?

Reply:
yes, I should be more consistent with typing for node pubkeys